src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -49,27 +49,27 @@ const InstructorCourses = ({ instructorID }) => {
       <Grid item xs={12}>
         <Grid className="accounts-table-heading" container>
           <Grid item xs={4}>
-            <Typography align="left" className="table-header">
+            <Typography variant="h4" align="left" className="table-header">
```

Comment:
Usually the rule of thumb I follow is if the props start getting a little long I give them their own line for better readability. Usually if there are 3 or more props. Not a huge concern but just something to keep in mind